probes/zrunner/runner.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zrunner
+
+import (
+	"github.com/ossf/scorecard/v4/checker"
+	serrors "github.com/ossf/scorecard/v4/errors"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes"
+)
+
+// Run runs the probes in probesToRun.
+func Run(raw *checker.RawResults, probesToRun []probes.ProbeImpl) ([]finding.Finding, error) {
+	var findings []finding.Finding
+	for _, item := range probesToRun {
+		f := item
```

Comment:
why not just rename the loop var? perhaps `probe` or `probeFunc`?

Reply:
oversight. Thank you.